src/Sylius/Component/Core/Inventory/Exception/NotEnoughUnitsOnHandException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Inventory\Exception;
+
+final class NotEnoughUnitsOnHandException extends \InvalidArgumentException
```

Comment:
```suggestion
final class NotEnoughUnitsOnHandException extends \RuntimeException
```
Not gonna argue but this seems more like a runtime exception rather than invalid argument

Reply:
In general I agree, but so far we have thrown out the `\InvalidArgumentException`, so not wanting to introduce a potential BC break, I wanted to leave the same type. Do you have a different opinion? 🤔

Reply:
Then add both, we can note in 1.14 that the invalid argument part will be removed in 2.0.